config.toml

```diff
@@ -0,0 +1,102 @@
+baseurl = "https://www.piqueserver.org/"
```

Comment:
baseurl should be left set to `""` to aid local development and make the site more portable.

Reply:
Would this get set when Travis builds then? Or will Hugo be fine with it being empty?

Reply:
I think it will be fine (all links will end up being `/path/to/file` instead of `https://www.piqueserver.org/path/to/file`. Not sure how to handle generated urls that *should* be absolute though, like canonical metadata urls or rss feed urls. I guess we can worry later.

Reply:
Although, not sure how it works through your test github.io domain. :thinking:

Reply:
My shower thoughts this morning thought that leaving it blank should work in all scenarios that this version of the website would deal with, but, like you say, stuff like RSS feeds would probably get broken. That'd probably be something we could have Travis edit before running Hugo if it was an issue down the road.